src/store/student-store.js

```diff
@@ -115,12 +127,12 @@ store.registerHandler('NEW_STUDENT', data => {
 
 store.registerHandler('NEW_CHECKOUT', data => {
     store.waitFor(CheckoutStore);
-    addItemsToStudent(data.studentId, data.equipmentAddresses);
+    addItemsToStudent(data.studentId, data.equipment, data.timestamp);
```

Comment:
`data.timestamp` is automatically added to all actions. I don't think we need to document this.